### DISCUSSION TITLE
part-ii/2020-83/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#287] [Question comments]
Part II 2020 Linear Analysis: Paper 2, Section II, I 

https://questions.tripos.org/part-ii/2020-83/

**Comment by stevezshen** [comment]
<details>
<summary>Spoiler warning (c)</summary>

Let $B = B_{\lVert\cdot\rVert}(0,1)$, $B' = B_{\lVert\cdot\rVert'}(0,1)$. $\forall x\in B'$, $\forall \phi\in \left(X,\lVert\cdot\rVert\right)^{*}$. $\text{ev}_{x}\left(\phi\right)<\infty$, so $\sup_{x\in B'}\lVert \text{ev}_x\rVert<\infty$, so $\sup_{x\in B'} \lVert x\rVert<\infty$. So $\exists C$ such that $\lVert \cdot \rVert' \leq C\lVert \cdot \rVert$.  

</details>
